controllers/front/CartController.php

```diff
@@ -432,6 +432,35 @@ protected function processChangeProductInCart()
                 'Shop.Notifications.Error'
             );
         }
+        
+        // Check minimal_quantity
+        if(!$this->id_product_attribute){
+            if($qty_to_check < $product->minimal_quantity){
+                 array_push(
+                     $this->{$ErrorKey},
+                     $this->trans(
+                         'The minimum purchase order quantity for the product %product% is %quantity%.',
+                         array('%product%' => $product->name, '%quantity%' => $product->minimal_quantity),
+                         'Shop.Notifications.Error'
+                     )
+                 );
+                 return;
+            }
+        }
+        else{
+             $combination = new Combination($this->id_product_attribute);
+             if($qty_to_check < $combination->minimal_quantity){
+                 array_push(
+                     $this->{$ErrorKey},
+                     $this->trans(
+                         'The minimum purchase order quantity for the product %product% is %quantity%.',
```

Comment:
Same here!

Reply:
Not required. But this also triggers if the customer adjust the quantity in the cart/checkout process. This way would return the product name to better identify the product. In case of slower systems the user could try to adjust more than one quantiy without waiting for the result. And since some other warnings have the product name. But agree, not required.

Reply:
Well, you're right, it could be useful when checking out if notified. Or it could just be displayed next to the item concerned? Currently, the quantity is blocked when on the product page, it is not blocked during the checkout. @marionf, can you confirm?